core/network/impl/synchronizer_impl.cpp

```diff
@@ -1406,4 +1408,51 @@ namespace kagome::network {
     ancestry_.clear();
     recent_requests_.clear();
   }
+
+  bool SynchronizerImpl::fetchHeadersBack(primitives::BlockNumber block,
+    CbResultVoid cb) {
+    BlocksRequest request{
+        BlockAttribute::HEADER,
+        block,
+        Direction::DESCENDING,
+        1,
```

Comment:
Request multiple headers
```suggestion
        std::nullopt,
```